### PR TITLE
Donnas first poem

### PR DESCRIPTION
**Pull request by cree007**

### CODE REVIEW

**Comment by cree007** [comment]
Adding my beautiful poem

File: coachDees-poem.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+I gather thoughts and share them here
```
Comment (jkschulz) [inline coachDees-poem.txt:2]: Can you add a title please?

**Review by jkschulz — CHANGES_REQUESTED** [review]
Looks great! One minor suggestion

File: coachDees-poem.txt (hunk 1 above)
Comment (cree007) [reply coachDees-poem.txt:2]: Great idea!

**Review by jkschulz — APPROVED** [review]

**Pull request merged**